environment.yml

```diff
@@ -13,12 +13,11 @@ dependencies:
   - cdo>=1.9.7
   - imagemagick
   - nco
+  - scikit-learn  # install from same installer as scipy; see issue ESMValTool: 1610
```

Comment:
It would be nice to have a reference to an issue from https://github.com/scikit-learn/scikit-learn/issues here, is there any?

Reply:
yes, bit of effort to dig through the issues, but here it is https://github.com/scikit-learn/scikit-learn/issues/14485

Reply:
looks to be hardware-related so if my HW got hit chances are others' may too

Reply:
Thanks for adding the info it! When that issue gets closed, we can remove scikit-learn from the conda environment again.